INTEGRATIONS-testnet-alpha.md

```diff
@@ -15,10 +15,14 @@ BTC P2WPKH deposit address for `nBTC` minting: `tb1qe60n447jylrxa96y6pfgy8pq6x9z
 
 ### Deepbook
 
-TODO
+Permissionless SUI/NBTC pool on deepbook-v3.1 on testnet:
+
+- Tx Digest: `D3XixtGXcDpPtbjcXqxSvQEkYid7r5Casj68hs3rPRiN`
+- Pool Object ID: `0x369978a9e522cc322020ff3183ab1f97a883ccc0e1b31fc6965a901e0006c9b4`
```

Comment:
is there any link to deepbook / magma that we can add here? ideally UI. If not, then we can do it later.

Reply:
i added link to magma. for deepbook there is no UI